src/test/java/no/unit/nva/model/contexttypes/DegreeTest.java

```diff
@@ -74,11 +74,11 @@ void objectMapperReturnsDegreeWhenInputIsValidJson(String seriesTitle,
     @DisplayName("Degree serializes expected json")
     @ParameterizedTest
     @CsvSource({
-            "A series title,123,Full publisher details,LEVEL_2,true,true,\"9780201309515|9788131700075\"",
-            ",, Full publisher details, LEVEL_2, true, true, \"9780201309515|9788131700075\"",
-            "A series title,,Full publisher details,LEVEL_2,true,true,\"9780201309515|9788131700075\"",
-            "Fulong of Oolong,12,T Publishing,LEVEL_0,false,false,\"9780201309515|9788131700075\"",
-            "A Marxist analysis of marking systems,6903,ACO,LEVEL_1,true,false,"
+            "A series title,123,Full publisher details,,true,true,\"9780201309515|9788131700075\"",
+            ",, Full publisher details,, true, true, \"9780201309515|9788131700075\"",
```

Comment:
Although I understand the practicality of the parametrized tests, it is not obvious what is the role of some of the examples. 
I understand that the 1st line contains all information (except for Level), the 2nd line is missing a title, the 3rd is missing the series number, the 5th is missing the ISBNs , but what is the 4th line missing?

Also I am not sure what is this test testing. 
Case A:
We are testing the JSON format i.e. new lines and spaces or is it testing the existence of the fields? If that is the case, why not test the configuration of the JSON mapper instead? Also what are the expectations? Are we expecting the empty fields to be non-existent or are we expecting them to be null? 

Case B:
We are testing the various cases where a Degree instance can be valid. i.e., A degree with all fields is filled is valid, A degree without title is valid, etc. 

Case C:
We are testing that the various valid formats of a degree can be serialized/de-serialized without data loss. 

The values of open access and peer review are never used in the Degree, and they are always set to false by the code (wrong usage of Boolean.getBoolean()),